codecs/text/src/main/java/com/datastax/oss/dsbulk/codecs/text/json/JsonNodeConvertingCodecProvider.java

```diff
@@ -391,6 +396,14 @@ public class JsonNodeConvertingCodecProvider implements ConvertingCodecProvider
                   nullStrings);
             case DATE_RANGE_CLASS_NAME:
               return new JsonNodeToDateRangeCodec(nullStrings);
+            case CQL_VECTOR_TYPE_CLASS_NAME:
+              CqlVectorType vectorType = (CqlVectorType) cqlType;
+              return new JsonNodeToCqlVectorCodec(
+                  vectorType,
+                  createJsonNodeConvertingCodec(vectorType.getSubtype(), codecFactory, false),
+                  TypeCodecs.FLOAT,
```

Comment:
Not a concern for now, but OSS C* is [planning on] adding support for vectors with many different types and not just `float`s [here](https://github.com/apache/cassandra/pull/2310)

Reply:
Worth noting: the rework of this functionality in the main PR for adding vector search to dsbulk does not make any assumptions as to the subtype of the input vector.